crates/email/src/account.rs

```diff
@@ -42,9 +43,9 @@ pub async fn send_password_reset_email(
 pub async fn send_verification_email(
   local_site: &LocalSite,
   user: &LocalUserView,
-  new_email: &str,
+  new_email: SensitiveString,
   pool: &mut DbPool<'_>,
-  settings: &Settings,
+  settings: &'static Settings,
```

Comment:
Should be the reference again, and you shouldn't need static there.

Reply:
Also needed for spawning the background task.